.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,12 +13,12 @@ I'm suggesting these edits to an existing topic:
 Please explain why these changes are necessary:
 
 
-************CURATING A NEW TOPIC PAGE************
+************CURATING A NEW TOPIC OR COLLECTION************
```

Comment:
Instead of modifying this to include collections, what about adding a separate `************CURATING A NEW COLLECTION************` below? Then lines 18-20 can be customized for collections specifically, and not try to cover both topics and collections. For example, when curating a new collection, this isn't right: `(e.g. https://github.com/topics/[NAME])`.